pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1476,6 +1476,16 @@
         "4.0.3.0": "4.4.0"
       }
     },
+    "System.Memory": {
+      "StableVersions": [
+      ],
+      "BaselineVersion": "4.4.0",
```

Comment:
Should the BaselineVersion be removed as well? It referrers to version that does not exist.